npm/qsharp/src/shared/circuit.ts

```diff
@@ -52,15 +71,17 @@ export interface Operation {
   /** Formatted gate arguments to be displayed. */
   displayArgs?: string;
   /** Nested operations within this operation. */
-  children?: Operation[];
+  children?: Operation[][];
+  /** Number of columns to span. */
+  columnWidth?: number;
   /** Whether gate is a measurement operation. */
-  isMeasurement: boolean;
+  isMeasurement?: boolean;
   /** Whether gate is a conditional operation. */
-  isConditional: boolean;
+  isConditional?: boolean;
   /** Whether gate is a controlled operation. */
-  isControlled: boolean;
+  isControlled?: boolean;
   /** Whether gate is an adjoint operation. */
-  isAdjoint: boolean;
+  isAdjoint?: boolean;
   /** Control registers the gate acts on. */
   controls?: Register[];
   /** Target registers the gate acts on. */
```

Comment:
For readability, can you put `targets` at the top below the other required properties? This is the most important property buried beneath all the optional ones.

Reply:
done